RELEASES.md

```diff
@@ -21,6 +21,38 @@ To summarize: **All our ibc-go releases allow chains to communicate successfully
 
 We ensure all major releases are supported by relayers ([hermes](https://github.com/informalsystems/ibc-rs), [rly](https://github.com/strangelove-ventures/relayer) and [ts-relayer](https://github.com/confio/ts-relayer) at the moment) which can relay between the new major release and older releases. We have no plans of upgrading to an IBC protocol specification v2.0, as this would be very disruptive to the ecosystem.
 
+## Release cycle
+
+IBC-Go follows a traditional release cycle involving an alpha, beta, and rc (release candidate) releases before finalizing a new version. As ibc-go works in a non-traditional area, we apply our own interpretation to each release type. We reserve the right to make both go API breaking changes and state machine breaking changes throughout the entire release cycle. The stable release guarentees do not go into affect until a final release is performed. 
+
+It is never advisable to use a non-final release in production. 
+
+### Alpha
+
+Alpha releases are intended to make avaliable new features as soon as they are functional. No correctness guarentees are made and alpha releases MAY contain serious security vulnerabilities, bugs, and lack of user tooling. Alpha releases may be made with known bugs and security vulnerabilities (so long as they don't affect the core functionality).
```

Comment:
I don't get what this means `Alpha releases may be made with known bugs and security vulnerabilities (so long as they don't affect the core functionality).` considering that the previous sentence already said `No correctness guarentees are made and alpha releases MAY contain serious security vulnerabilities, bugs, and lack of user tooling`

Reply:
updated